collection.css

```diff
@@ -0,0 +1,78 @@
+body {
```

Comment:
add this in to make sure box sizing goes ok:
* {
    box-sizing: border-box;
}